### PR TITLE
Memo sorting fix

### PR DESCRIPTION
**Pull request by niyid**
Fixes issue #1289

### CODE REVIEW

File: desktop/src/main/java/haveno/desktop/main/funds/transactions/TransactionsView.java [hunk 1]
```diff
@@ -162,7 +162,9 @@ public void initialize() {
         transactionColumn.setComparator(Comparator.comparing(TransactionsListItem::getTxId));
         amountColumn.setComparator(Comparator.comparing(TransactionsListItem::getAmount));
         confidenceColumn.setComparator(Comparator.comparingLong(TransactionsListItem::getNumConfirmations));
-        memoColumn.setComparator(Comparator.comparing(TransactionsListItem::getMemo));
+        memoColumn.setComparator(
+    Comparator.comparing(TransactionsListItem::getMemo, Comparator.nullsLast(Comparator.naturalOrder()))
```
Comment (woodser) [inline desktop/src/main/java/haveno/desktop/main/funds/transactions/TransactionsView.java:166]: Please keep this on one line, for consistency with the surrounding formatting.

**Comment by woodser** [comment]
Awaiting this to be moved to one line so it can be merged.

File: desktop/src/main/java/haveno/desktop/main/funds/transactions/TransactionsView.java (hunk 1 above)
Comment (niyid) [reply desktop/src/main/java/haveno/desktop/main/funds/transactions/TransactionsView.java]: Done. Thanks for your patience.

Comment (woodser) [reply desktop/src/main/java/haveno/desktop/main/funds/transactions/TransactionsView.java]: It's still on 2 lines. Please keep to one line like the surrounding formatting.

**Pull request merged**